weed/topology/topology_vacuum.go

```diff
@@ -138,6 +139,8 @@ func (t *Topology) batchVacuumVolumeCommit(grpcDialOption grpc.DialOption, vl *V
 		} else {
 			glog.V(0).Infof("Complete Committing vacuum %d on %s", vid, dn.Url())
 		}
+		// small space out for the same volume id https://github.com/seaweedfs/seaweedfs/issues/3369
+		time.Sleep(3 * time.Second)
```

Comment:
this is a separate issue. How to do it needs discussion. The value would not work for large clusters.

Reply:
ok